wikimedia/wikipedia-ios
# Configure iOS schemas to send data to MEP, remove legacy schemas

**Phabricator:** https://phabricator.wikimedia.org/T327341 and https://phabricator.wikimedia.org/T331481

### Notes
* This PR sets our app to send analytics data to the Modern Events Platform.
* It removes the base classes that served the legacy system, removes some funnels that were not to be moved into MEP, and updates the existing funnels to work with MEP
* I tried to merge most of the deletion commits, but the ones that were too far apart on the timeline were causing conflicts,  so there are some separate commits with deletions

### Test Steps
1. Run the app. The schemas for the explore feed, funnel, settings, login, readings lists, and user history (that captures a broader picture of the app's current state) were updated, so please interact with all these areas.
2. Check if info is being sent on a network debugger or the events platform.
3. A lot of code was removed, so a broader smoke test would be good while exploring the app
